doc/man/man3/crypto_sign_init_first_pass.3monocypher

```diff
@@ -167,17 +167,18 @@ These functions implement EdDSA with Curve25519 and Blake2b.
 This is the same as Ed25519, with Blake2b instead of SHA-512.
 Ed25519 is described in RFC 7748.
 .Sh SECURITY CONSIDERATIONS
-In addition to those documented on
-.Xr crypto_sign 3monocypher ,
-incremental signature verification encourages the user to start
-processing the message before verification is done.
+Messages are not verified until the call to
+.Xr crypto_check_final .
```

Comment:
Did you mean `.Fn` (function name) instead of `.Xr` (external reference)?

Reply:
whoops